staging/src/k8s.io/apimachinery/pkg/runtime/serializer/codec_factory.go

```diff
@@ -147,6 +149,12 @@ func WithSerializer(f func(runtime.ObjectCreater, runtime.ObjectTyper) runtime.S
 	}
 }
 
+func WithStreamingCollectionEcoderToJson() CodecFactoryOptionsMutator {
```

Comment:
touch up function name please? `WithStreamingCollectionEncoderToJson` (missing an `n` in Encoder)

Reply:
`WithStreamingCollectionEncoderToJSON` would be good too

Reply:
Done